Gemfile

```diff
@@ -52,6 +52,7 @@ group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'debug', platforms: %i[mri windows]
   gem 'factory_bot_rails'
+  gen 'rails-controller-testing'
```

Comment:
- [ ] You can see from the highlighted point, this gem notation is spelled as `gen`. Kindly fix this 👍 
- After the above is implemented, kindly make sure to run the test and it will be cross-checked after your changes. Good job 👏